backend/tests/e2e/test_api_e2e.py

```
@@ -0,0 +1,988 @@
+            test_settings.default_llm_provider = "gemini"
+        finally:
+            # Restore original env vars
+            for key, original_value in original_env.items():
+                if original_value is None:
+                    os.environ.pop(key, None)
+                else:
+                    os.environ[key] = original_value
+        
+        return test_settings
+
+    async def _validate_comprehensive_api_data(self, test_client, alert_id: str, session_id: str):
```

Ruff (0.12.2)

Unused method argument: `alert_id`

(ARG002)